src/test/java/tests/yandex_email/BaseStep.java

```diff
@@ -0,0 +1,34 @@
+package tests.yandex_email;
+
+import com.google.common.collect.ImmutableMap;
+import driver.Config;
+import driver.DriverSingleton;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.openqa.selenium.WebDriver;
+
+import static com.github.automatedowl.tools.AllureEnvironmentWriter.allureEnvironmentWriter;
+
+public class BaseStep {
+    static WebDriver driver;
+
+    @BeforeAll
+    public static void initDriver(){
+        driver = DriverSingleton.getInstance().getDriver(Config.CHROME);
+    }
+    @BeforeAll
+    public static void setAllureEnvironment() {
+        allureEnvironmentWriter(
+                ImmutableMap.<String, String>builder()
+                        .put("Browser", "Chrome")
```

Comment:
why do you set up these params explicitly? and what about FF

Reply:
fixed